test/ui-test/testSuites/suite_communities/tst_communityNavigation/test.feature

```diff
@@ -0,0 +1,36 @@
+#******************************************************************************
+# Status.im
+#*****************************************************************************/
+#/**
+# * \file	test.feature
+# *
+# * \test	Status Desktop - Community
+# * \date	January 2023
+# **
+# *****************************************************************************/
+
+Feature: Status Desktop community navigation
+
+    As an admin user I want to create a community and be able to leave it.
+
+   	The feature start sequence is the following (setup on its own `bdd_hooks`):
+    ** given A first time user lands on the status desktop and generates new key
+    ** when user signs up with username "tester123" and password "TesTEr16843/!@00"
+    ** and the user lands on the signed in app
+
+    Scenario Outline: User creates and leaves community
+      # Create a community
+      Given the user opens the community portal section
+      And the user creates a community named <community_name>, with description "My community description", intro "Community Intro" and outro "Community Outro"
```

Comment:
We usually do like this here (it makes the statement more readable in case this statement is used with a direct string here): 
```suggestion
      And the user creates a community named "<community_name>", with description "My community description", intro "Community Intro" and outro "Community Outro"
```

Reply:
I actually did this intentionally, relying on the syntax highlighter.

Take a look:
<img width="500" alt="image" src="https://user-images.githubusercontent.com/25482501/214786804-80308dcf-d752-4783-b7a4-5c41e4909064.png">

Note that the variable without quotes is highlighted. 
And therefore it's kinda easier to read the code.

This is my own opinion and I'm not insisting 🙂 
I'll fix it to make it as everywhere, but we can maybe have a call to discuss this 🙃